docs/releases/v10/version-v10.0.0.md

```diff
@@ -34,7 +34,7 @@ Custom fields will be displayed at the bottom of the community's settings page.
 
 ![](imgs/custom_fields_communities.png)
 
-When a community field is required, it will be also displayed in the form to [create a new community](https://inveniordm.web.cern.ch/communities/new).
+When a community field is required, it will be also displayed in the form to [create a new community](https://inveniordm.web.cern.ch/communities-new).
```

Comment:
I don't think it makes sense to update the docs for v10, since at the point, the endpoint was valid (and based on https://github.com/inveniosoftware/invenio-communities/pull/1210#discussion_r1827988925 it should continue to be).

Reply:
Alright I see, thanks!